SU-SWS/decanter
# Adding global footer variants and link tweaks for legibility, change layout at MD breakpoint to match Figma

# READY FOR REVIEW

# Summary
- Add two global footer variants with bright red and soft black backgrounds. New variants have passed color contrast test.
- Remove all `font-size: 300;` in global footer links since they are not legible in some browsers (e.g., firefox). Tweak font weight, font smoothing and spacing between links so they are more legible and still looks good.
- Use center aligned layout (same as at `sm` breakpoint) for `md` breakpoint to give it more breathing room. Increase spacing between menu links.
- Remove the opacity 0.9 on the global footer links so they're more legible.
- Add `aria-label="global footer menu"` to the nav element (AMP scan result)
- Use the logo component twig for the logo in the global footer.

# Needed By (Date)
- Whenever

# Urgency
- Not urgent

# Steps to Test

1. Pull this branch and compile styleguide.
2. Go to the Component -> Global Footer section and check that the two new variants are there. 
3. Check every breakpoint and make sure menus and text that are meant to be on one line don't wrap to 2nd line. Check that everything is legible in all modern browsers.

# Affected Projects or Products
- Decanter
- Everything that uses Decanter

# Associated Issues and/or People
- #499 
- The new variants are approved by Dana Granoski, UComm's Brand Designer.
